gradle/developer-config.gradle

```diff
@@ -17,9 +17,10 @@ task accessToken {
                 "    <!-- Your valhalla url (example: https://valhalla1.openstreetmap.de/route) -->\n" +
                 "    <!-- Don't use the following server in production, it is for demonstration purposes only: -->\n" +
                 "    <string name=\"valhalla_url\" translatable=\"false\">https://valhalla1.openstreetmap.de/route</string>\n" +
-                "    <!-- Instead of valhalla you can use GraphHopper for the path finding. Example: https://graphhopper.com/api/1/navigate?key=YOUR_API_KEY or a local server -->\n" +
+                "    <!-- Instead of valhalla you can use GraphHopper for the path finding. -->\n" +
                 "    <!-- Don't use the following API key in production, it is for demonstration purposes only: -->\n" +
-                "    <string name=\"graphhopper_url\" translatable=\"false\">https://graphhopper.com/api/1/navigate?key=7088b84f-4cee-4059-96de-fd0cbda2fdff</string>\n" +
+                "    <!-- <string name=\"graphhopper_key\" translatable=\"false\">7088b84f-4cee-4059-96de-fd0cbda2fdff</string> -->\n" +
```

Comment:
I think you can't have these commented out, or the build will fail (as it is right now)